docs/HowTo/Configure-TLS.md

```diff
@@ -0,0 +1,118 @@
+---
+description: Secure communication using TLS
+---
+
+# Configure TLS
+
+Configure TLS communication from the command line. Clients and servers
+communicate using HTTP JSON-RPCs.
+
+**Prerequisites**:
+
+* EthSigner's password-protected PKSC #12 keystore.
+* File containing the keystore password.
+
+## Client TLS Connection
+
+Allow clients (for example a dApp, or curl) to send and receive secure
+HTTP JSON-RPCs.
+
+**Prerequisites**:
+
+* The client must be configured for TLS.
+* Client's PKSC #12 keystore information.
+
+
+### Create the Known Clients File
+
+Create a file (in this example, `knownClients`) that lists one or more clients
+that are trusted to connect to EthSigner. The file contents use the format
+`<common_name> <hex-string>` where:
+
+* `<common_name>` is the Common Name used for the client's keystore
+* `<hex-string>` is the SHA-256 fingerprint of the client's keystore.
+
+!!! example
+    ```
+    curl_client DF:65:B8:02:08:5E:91:82:0F:91:F5:1C:96:56:92:C4:1A:F6:C6:27:FD:6C:FC:31:F2:BB:90:17:22:59:5B:50
+    ```
+    
+You can use openssl or keytool to display the fingerprint. For example:
+
+!!! example
+    ```
+    keytool -list -v -keystore <keystore> -storetype PKCS12 -storepass <password>
+    ```
+    
+### Start EthSigner
+
+```bash
+ethsigner --tls-keystore-file=/Users/me/my_node/keystore.pfx --tls-keystore-password-file=/Users/me/my_node/keystorePassword --tls-known-clients-file=/Users/me/my_node/knownClients  --tls-allow-ca-clients
+```
+
+The command line:
+
+* Specifies the EthSigner keystore using the [`--tls-keystore-file`](../Reference/CLI/CLI-Syntax.md#tls-keystore-file) option.
+* Specifies the file that contains the password to decrypt the
+keystore using the [`--tls-keystore-password-file`](../Reference/CLI/CLI-Syntax.md#tls-keystore-password-file) option.
+* [Specify the clients](#create-the-known-clients-file) that are trusted to connect to EthSigner using the
+[`tls-known-clients-file`](../Reference/CLI/CLI-Syntax.md#tls-known-clients-file) option.
+* Allow access to clients with trusted CA certificates using the [`--tls-allow-ca-clients`](../Reference/CLI/CLI-Syntax.md#tls-allow-ca-clients) option.
+
+!!! note
+    Use the [`--tls-allow-any-client`](../Reference/CLI/CLI-Syntax.md#tls-allow-any-client) option to allow
+    access to any client that presents a certificate. 
```

Comment:
@rain-on can confirm, but I think when specifying `--tls-allow-any-client`, the client doesn't need to present a certificate at all (i.e. we disable client auth). If it presents, I believe we ignore it if this option is set.

Reply:
Confirmed with Trent, the client does not need to present a certificate.